ydb/docs/en/core/reference/languages-and-apis/ado-net/connection_parameters.md

```diff
@@ -0,0 +1,49 @@
+# Connection parameters {#connection-parameters}
```

Comment:
Mention ADO.NET in each article name. It helps people understand what they are reading without looking at the sidebar, as well as with SEO.